bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -155,7 +155,7 @@ metadata:
     description: OADP (OpenShift API for Data Protection) operator sets up and installs
       Data Protection Applications on the OpenShift platform.
     olm.skipRange: '>=0.0.0 <99.0.0'
-    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/cluster-monitoring: "false"
```

Comment:
Shouldn't we remove this annotation altogether, instead of marking it as false ?

Reply:
@shubham-pampattiwar 

I am not sure about that. I always prefer to explicitly tell what is the intended behavior as:
 - current the defaults may change over time
 - I don't know how the update/upgrade scenario for existing namespaces will behave and here we explicitly say what is the intended label for that setting, which can be reflected better in the documentation to check for the false or true on that label, otherwise we also need to check if that label exists
 - user have knowledge by looking at the label that the cluster-monitoring is disabled, so a query on the cluster based on labels can quickly identify the ones where cluster-monitoring is enabled (true) but also negative filter is nice to have, or at least doesn't hurt

let me know what do you think as I am happy to remove that in case you disagree

Reply:
I would probably remove the label as well to be safe and future-proof. but just my two cents it's a nit IMO.

Reply:
Ok I will propose change to this PR and remove the labels.

Reply:
New revision to that PR removed all *cluster-monitoring* labels and also part that was adding this for the disconnected environment.

@shawn-hurley @shubham-pampattiwar